### PR TITLE
Feature/add jest testing

### PR DESCRIPTION
**Pull request by MrRamoun**
- ✔️ Set up React Testing Library.
- ✔️ Follow the instructions in the official documentation.
- ✔️ Write unit tests for the files `operate.js` and `calculate.js` using Jest.
- ✔️ Create unit tests for all React components.
- ✔️ Use Jest snapshots to test the components.
- ✔️ Use React Testing Library to simulate user interaction.
- ✔️ Run your full tests suite using Jest. All tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,54 +1,64 @@
...
-
-Give a ⭐️ if you like this project!
-
-## 📝 License
-
-This project is [MIT](./MIT.md) licensed.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Math Magicians
+
```
Comment (TedLivist) [inline README.md:4]: - [ ] For proper documentation, in your README kindly add the command for running the tests.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @MrRamoun and @fherrerao,

Good job so far!
✔️ Use of React Testing Library
✔️ Tests cover most operations
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BrianSammit — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello team 👋, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional 
- Your repo is professional 
- Well structured files 


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

Well-Done-Applause


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (MrRamoun) [reply README.md:4]: Thank you so much 👍

**Pull request merged**